pyan/analyzer.py

```diff
@@ -387,6 +397,14 @@ def visit_ImportFrom(self, node):
             # with node.level).
             #
             # https://greentreesnakes.readthedocs.io/en/latest/nodes.html?highlight=functiondef#ImportFrom
+            # pyan can handel Relative imports such as "from .mod import foo" and "from ..mod import foo"
+            if node.level != 0:
+                self.logger.debug("Old ImportFrom: from %s import %s, %s:%s" % (node.module, [format_alias(x) for x in node.names], self.filename, node.lineno))
+                tgt_level = node.level 
+                current_module_namespace = self.module_name.rsplit('.', tgt_level)[0]
+                node.module = current_module_namespace+'.'+node.module
+                self.logger.debug("New ImportFrom: from %s import %s, %s:%s" % (node.module, [format_alias(x) for x in node.names], self.filename, node.lineno))
+
```

Comment:
Please use the PEP8 convention of exactly one whitespace on both sides of a binary operator: `current_module_namespace + '.' + node.module`.

Same comment as above about modifying `node`.

Also, debug messages need some thinking to maximize readability of the logs, due to the different possibilities here.

If `node.level == 0`, this section doesn't trigger, so the "ImportFrom (resolved):" message further above remains the final one printed.

But if `node.level != 0`, these messages print, too.